modules/administration_manual/pages/release_notes.adoc

```diff
@@ -724,17 +689,15 @@ If this is not possible (e.g. no internet connection or clustered setup) you wil
 (`occ app:disable templateeditor`) or
 xref:installation/apps_management_installation.adoc#manually-installing-apps[download and install it manually].
 
-[[10.0.8-solved-known-issues]]
 === Solved known issues
 
 * Bogus ``Login failed`` log entries have been removed (see
```

Comment:
I think you can use single back ticks instead double once.

Reply:
Well said, but that's a change for a separate PR.

Reply:
Yes, we could. But I believe, strongly, that PRs are as small and as specific as possible, hence the request for a separate PR.